uis/src/com/biglybt/ui/swt/devices/DevicesFTUX.java

```diff
@@ -143,12 +143,12 @@ void open(Runnable fire_on_install) {
 		}
 
 		Link lblLearnMore = new Link(shell, SWT.NONE);
```

Comment:
Can this be substituted with LinkLabel ?

Reply:
Probably :)  I'm sure there's also other places that aren't using LinkLabel.
I have some pending changes to the ConfigSection classes that fix a few other non-LinkLabel usage